plugin/output/s3/s3.go

```diff
@@ -215,6 +218,10 @@ type Config struct {
 	// > DynamicBucketsLimit regulates how many buckets can be created dynamically.
 	// > Prevents problems when some random strings in BucketEventField where
 	DynamicBucketsLimit int `json:"dynamic_buckets_limit" default:"32"` // *
+	// > @3@4@5@6
+	// > UploadTimeout sets max upload timeout to s3.
+	UploadTimeout  cfg.Duration `json:"upload_timeout" default:"1m" parse:"duration"`
```

Comment:
Comment should looks like:
```
	// > @3@4@5@6
	// > 
	// > Sets upload timeout.
```

Empty line is missing. Don't repeat parameter name. Also please format other parameters as well.

Reply:
Fixed.